git_setup_windows/git_setup_windows.md

```diff
@@ -0,0 +1,339 @@
+| 4. Click "Next" in the lower right. | ![Select Start Menu Folder screen](media/05select_folder.png)<!-- style = "max-width:400px;" border = 5px solid --> |
+| 5. If you have a favorite text editor such as Atom, go ahead and select that. You can scroll up for more options like `nano`, or select Notepad, which will feel the most familiar if you are used to word processing programs. | ![Choosing the default editor screen. The "Use Notepad as Git's default editor" option is selected from the dropdown menu.](media/06default_editor.png)<!-- style = "max-width:400px;" border = 5px solid -->|
+| 6. On this screen you want to change the default branch name to "main" to match with the conventions of GitHub. | ![Branch naming screen. The second option, "Override the default branch name for new responses" is selected and the word "main" is entered into the text box.](media/07branch_naming.png)<!-- style = "max-width:400px;" border = 5px solid -->|
+| 7. Allowing 3rd parties, such as GitHub Desktop, may be extremely useful to you in the future, so we recommend selecting this option. | ![Adjusting your PATH screen. The second option, "Git from the command line and also from 3rd-party software" is selected](media/08path.png)<!-- style = "max-width:400px;" border = 5px solid -->|
+| 8. You can click "Next" on each of the subsequent screens. | ![SSH screen](media/09ssh.png)<!-- style = "max-width:400px;" border = 5px solid -->|
+| 9. Click "Next" in the lower right of the screen.| ![HTTPS screen](media/10https.png)<!-- style = "max-width:400px;" border = 5px solid -->|
+| 10. Click "Next" in the lower right of the screen.| ![Configuring line ending conversions screen](media/11line_endings.png)<!-- style = "max-width:400px;" border = 5px solid -->|
+| 11. Click "Next" in the lower right of the screen.| ![Configuring terminal emulator screen](media/12terminal_emulator.png)<!-- style = "max-width:400px;" border = 5px solid -->|
+| 12. Click "Next" in the lower right of the screen.| ![Behavior of `git pull` screen](media/13git_pull.png)<!-- style = "max-width:400px;" border = 5px solid -->|
+| 13. Click "Next" in the lower right of the screen.| ![Choose a credential helper screen](media/14credential.png)<!-- style = "max-width:400px;" border = 5px solid -->|
+| 14. Click "Next" in the lower right of the screen.| ![Extra options screen](media/15extra_options.png)<!-- style = "max-width:400px;" border = 5px solid -->|
+| 15. Click "Next" in the lower right of the screen.| ![Experimental options screen](media/16experimental_options.png)<!-- style = "max-width:400px;" border = 5px solid -->|
+| 16. Click "Finish" to complete the installation.|  ![Final screen, select "Finish" in the lower right to complete setup](media/17finish.png)<!-- style = "max-width:400px;" border = 5px solid -->|
+
+
+### Git Bash
+
+You should now be all ready to use Git on your machine. Open up Git Bash to use for the rest of this module.
+
+| Description | Screenshot |
```

Comment:
I was having a lot of trouble with the combo of screenshots and text rendering in a nice readable way (lists were getting really funky, and I find one or two lines of text between pictures to be really hard to read). I'm definitely open to  suggestions of better ways to do it. I could also shrink the images a bit so that there is more space for the words in the table? Maybe making each one thumbnail size and clickable to see the larger image?

Reply:
This is maybe an iffy solution, but could you just drop the first screenshot here? I feel like they should be able to use the Start Menu to find an application on their own. Then there would be just the one screenshot for this section, and it wouldn't be orphaned lines of text between images. You could also scale the image down a bit, of course, if you feel like it's overwhelming the small amount of text.

Reply:
I like it, yeah it seems reasonable to think they can find it in the start menu.